### DISCUSSION TITLE
How to check for existing Queue schedulers present on a queue?

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [taskforcesh/bullmq#1072] [Q&A] [answered]
I am using a single container for both the queues and queues schedulers, but this leads to many queue schedulers created if scaling horizontally. Is there a way to check if a queue has a queue scheduler attached? That way, my code would only create a queue scheduler if a certain number are not already present

**Comment by manast** [comment, marked as answer]
This is a great use case, basically what we need is a getSchedulers() method that we can call on the Queue instance. If you create an issue as a feature request I think we could att this pretty quick.

**Reply by bilalshaikh42** [reply]
Thanks! Done as https://github.com/taskforcesh/bullmq/issues/1075
